ddtrace/internal/datadog/profiling/cmake/FindCppcheck.cmake

```diff
@@ -18,11 +18,6 @@ if (DO_CPPCHECK)
     )
     set(CPPCHECK_EXECUTABLE ${CMAKE_BINARY_DIR}/cppcheck/bin/cppcheck)
   endif()
-
-  # The function we use to register targets for cppcheck would require us to run separate
-  # commands for each target, which is annoying.  Instead we'll consolidate all the targets
-  # as dependencies of a single target, and then run that target.
-  add_custom_target(cppcheck_runner ALL COMMENT "Runs cppcheck on all defined targets")
```

Comment:
Why is this removed?

Reply:
Ah, because it's not used.  Later in the cmake you'll see the true target we actually use.  At one point I factored this differently and just didn't remove the original runner.
```
  # Create a standard target to run everything
  if (NOT TARGET cppcheck)
    add_custom_target(cppcheck COMMENT "Runs cppcheck on all projects")
  endif()
  if (DO_CPPCHECK)
    add_dependencies(cppcheck ${NAME})
  endif()
  ```